source/api/content-state/0.9/index.md

```diff
@@ -158,7 +158,7 @@ The content state _MAY_ be supplied as JSON-LD, as the value of the `target` pro
 }
 ```
 
-This form is better suited to scenarios where compactness is important, for example a query string parameter, where it is will be combined with base64url encoding as described in Section 2.3.
+This form is better suited to scenarios where compactness is important, for example a query string parameter, where it is will be combined with the encoding described in Section 2.3.
```

Comment:
Should it say "it is combined" or "will be combined"?

Reply:
good catch, have pushed a fix